index.html

```diff
@@ -5,11 +5,375 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Medical care</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="/css/index.css" />
   </head>
   <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header class="header">
+      <nav class="nav">
+        <h5 class="nav__logo">Medical Care</h5>
+        <ul class="nav-list nav--default">
+          <li class="nav__item"><a href="/#" class="nav__link">Home</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Products</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Pricing</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="header-action">
+        <button class="button button--bg-white nav--default">Login</button>
+        <button
+          class="button button--bg-green nav--default button--after-icon-arrow"
+        >
+          Join Us
+        </button>
+        <button class="nav--mobile header-action__menu-icon">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            class="w-6 h-6"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+            />
+          </svg>
+        </button>
+      </div>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="hero__content">
+          <h3 class="title">Welcome</h3>
+          <h5 class="headline headline--size-large">
+            A Great Place <br />
+            to Receive Care
+          </h5>
+          <p class="description description--size-large">
+            Get your best looking smile now!
+          </p>
+          <div class="hero-action">
```

Comment:
I would make this `div` an element of `hero` block, thereby changing from `hero-action` to `hero__action`